frontend/src/components/QuestionOverview.tsx

```diff
@@ -40,23 +50,20 @@ export default function QuestionOverview(props: QuestionOverviewProps) {
             <label>
                 <h4>2- Which weather do you prefer?</h4>
                 <select
-                       onChange={onQuestionChange} value={questions}
-                    // onChange={(event) => setQuestion(event.target.value)}
-
+                    onChange={(event) => setQuestionWeather(event.target.value)}
+                    defaultValue={""}
                 >
                     <option value={""} disabled={true}>Select one answer</option>
                     <option value={"SUNNY"}>SUNNY</option>
                     <option value={"RAINY"}>RAINY</option>
                     <option value={"SNOWY"}>SNOWY</option>
                 </select>
             </label>
-            <br/>
-
-
 
             <br/>
```

Comment:
Linebreaks finde ich immer schwierig in html, kann sein das es später wenn es auf dem Handy angezeigt wird Probleme macht... Würde versuchen das über css zu regeln...